controllers/metering/api/v1/metering_types.go

```diff
@@ -94,3 +113,18 @@ type MeteringList struct {
 func init() {
 	SchemeBuilder.Register(&Metering{}, &MeteringList{})
 }
+
+func DefaultResourceQuota() corev1.ResourceList {
+	return corev1.ResourceList{
+		//corev1.ResourceRequestsCPU:    resource.MustParse("100"),
+		corev1.ResourceLimitsCPU: resource.MustParse("10000"),
+		//corev1.ResourceRequestsMemory: resource.MustParse("100"),
+		corev1.ResourceLimitsMemory: resource.MustParse("10000Gi"),
```

Comment:
Too large for test purpose?

Reply:
<img width="666" alt="image" src="https://user-images.githubusercontent.com/87080562/209899079-bc1e8324-34c7-4bee-8a01-ae2ff1247a36.png">